src/Firebase/base-firebase-integration.ts

```diff
@@ -1,12 +1,16 @@
 import { initializeApp } from "firebase/app";
-import { getAnalytics, logEvent } from "firebase/analytics";
+import { getAnalytics, logEvent ,setUserProperties} from "firebase/analytics";
 import { firebaseConfig } from "./firebase-config";
+import { source, campaign_id } from "@common";
 
 export class BaseFirebaseIntegration {
     firebaseApp: any;
     analytics: any;
     constructor() {
         this.initializeFirebase();
+        console.log(" source : ",source ," and  campaign_id: ",campaign_id);
```

Comment:
@ashwinnair-chimple ,
Why I'm getting these file changes?
We already merged these changes to main last time and also I have confirmed that these changes exists on main branch already.

Reply:
@Nikhil77sutara go to backup-main branch take pull from main branch and then raise PR this is cause the back-main is not yet updated with main branch